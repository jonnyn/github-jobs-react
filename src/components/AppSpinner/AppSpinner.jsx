import React from 'react'
import { Container, Spinner } from 'react-bootstrap';

const AppSpinner = () => (
    <Container fluid="md">
        <div className="d-flex justify-content-center">
            <Spinner animation="border" role="status" variant="primary">
                <span className="sr-only">Loading...</span>
            </Spinner>
        </div>
    </Container>
);

export default AppSpinner;