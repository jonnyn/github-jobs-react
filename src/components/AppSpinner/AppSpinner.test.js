import React from 'react';
import AppSpinner from './AppSpinner';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const component = renderer.create(<AppSpinner />,);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
