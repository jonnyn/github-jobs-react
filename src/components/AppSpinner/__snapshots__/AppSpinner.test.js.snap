// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container-md"
>
  <div
    className="d-flex justify-content-center"
  >
    <div
      className="spinner-border text-primary"
      role="status"
    >
      <span
        className="sr-only"
      >
        Loading...
      </span>
    </div>
  </div>
</div>
`;
